build.sbt

```diff
@@ -25,20 +31,54 @@ def jsonDependencies(scalaVersion: String) = Seq(
   logback % Test
 ) ++ jacksons ++ specsBuild.map(_ % Test)
 
+// Common settings 
+import com.typesafe.sbt.SbtScalariform._
+import scalariform.formatter.preferences._
+
+val previousVersion = Some("2.6.0-M1") // first from this separate repo
```

Comment:
@cchantep Name changes could be handled by applying broader filters, for example, to filter out `ValidationError` issues, we can get pretty close by using `{ problem => !problem.ref.toString.contains("ValidationError") }`.

If I understand correctly, the goal is to document the things that changed in this version, so it's much less useful if we use a milestone version.

Or we can just switch to `None` and not worry about it until we have a better solution. I don't think MiMa is well designed for this use case.